file/random_access_file_reader.cc

```diff
@@ -142,6 +142,9 @@ IOStatus RandomAccessFileReader::Read(const IOOptions& opts, uint64_t offset,
           auto finish_ts = FileOperationInfo::FinishNow();
           NotifyOnFileReadFinish(orig_offset, tmp.size(), start_ts, finish_ts,
                                  io_s);
+          if (!io_s.ok()) {
+            NotifyOnIOError(io_s, "Read", file_name(), tmp.size(), orig_offset);
```

Comment:
Is it possible to use enum for the "Read", "Fsync" "Flush"? That will easier to track

Reply:
There is a ```FileOperationType``` in listener.h that can be used here.